app/domain/repository/persist_repository_from_upstream.rb

```diff
@@ -111,13 +111,16 @@ def self.correct_status_from_upstream(repository, archived_upstream:)
                             })
       # set to unmaintained if we do not have another status already assigned
       "Unmaintained"
-    elsif !archived_upstream && repository.status == "Unmaintained"
-      StructuredLog.capture("REPOSITORY_REMOVE_UNMAINTAINED_STATUS",
-                            {
-                              repository_host: repository.host_type,
-                              repository_id: repository.id,
-                              full_name: repository.full_name,
-                            })
+    elsif !archived_upstream && %w[Unmaintained Removed].include?(repository.status)
+      if repository.status == "Unmaintained"
```

Comment:
🎱 Thoughts about removing the nested conditional and just logging on both, something like `REPOSITORY_UNSETTING_REMOVED_STATUS` with a `current_status` key/value? If we're interested in when unmaintained gets removed, we could also be interested in when the removed status gets removed too.